### ISSUE TITLE
Configure external Custom Action Server

### ISSUE THREAD

**Issue by ferenc-hechler** [RasaHQ/rasa-x-helm#174]
This is a feature request.

As described [here](https://rasa.com/docs/rasa-x/installation-and-setup/customize#adding-a-custom-action-server) it is possible to deploy a custom action server by setting the docker image and tag in the values.yml "app.name" and "app.tag".

Instead of deploying a given docker image we would like to configure the custom action server endpoint to use an already existing action server. So, in the values.yml it could look like this:
```
app:
  install: false
  existingUrl: http://my-action-server.namespace.svc:5055/webhook
```

Then the "XXX-rasa-x-app" action server should not be created and in the configmap "XXX-rasa-configuration-files" the entry "rasa-endpoints.action_endpoint.url" should be set to the value given in ".Values.app.existingUrl".

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @degiz will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by ferenc-hechler** [comment]
I created pull request https://github.com/RasaHQ/rasa-x-helm/pull/185 for this issue. Local tests were successfull.

**Comment by tczekajlo** [comment]
Resolved by #185

**Comment by ferenc-hechler** [comment]
THX for your support @tczekajlo
